jep/0000/README.adoc

```diff
@@ -0,0 +1,225 @@
+= JEP-0000: External Artifact Storage
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.**JEP Template**
+
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 0000
```

Comment:
Number should be assigned

Reply:
@oleg-nenashev The editor sets the number unless the sponsor requests a specific one.